README.rst

```diff
@@ -385,51 +417,209 @@ Add ``openwisp_utils.loaders.DependencyLoader`` to ``TEMPLATES`` in your ``setti
+
+9. Create database migrations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create and apply database migrations::
+
+    ./manage.py makemigrations
+    ./manage.py migrate
+
+For more information, refer to the
+`"Migrations" section in the django documentation <https://docs.djangoproject.com/en/dev/topics/migrations/>`_.
+
+10. Create the admin
+~~~~~~~~~~~~~~~~~~~~
+
+Refer to the `admin.py file of the sample app <https://github.com/openwisp/openwisp-firmware-upgrader/tree/master/tests/openwisp2/sample_firmware_upgrader/admin.py>`_.
+
+To introduce changes to the admin, you can do it in two main ways which are described below.
+
+**Note**: for more information regarding how the django admin works, or how it can be customized,
+please refer to `"The django admin site" section in the django documentation <https://docs.djangoproject.com/en/dev/ref/contrib/admin/>`_.
+
+1. Monkey patching
+##################
+
+If the changes you need to add are relatively small, you can resort to monkey patching.
+
+For example:
+
+.. code-block:: python
+
+    from openwisp_firmware_upgrader.admin import (  # noqa
+        BatchUpgradeOperationAdmin,
+        BuildAdmin,
+        CategoryAdmin,
+    )
+
+    BuildAdmin.list_display.insert(1, 'my_custom_field')
+    BuildAdmin.ordering = ['-my_custom_field']
+
+2. Inheriting admin classes
+###########################
+
+If you need to introduce significant changes and/or you don't want to resort to
+monkey patching, you can proceed as follows:
+
+.. code-block:: python
+
+    from django.contrib import admin
+    from openwisp_firmware_upgrader.admin import (
+        BatchUpgradeOperationAdmin as BaseBatchUpgradeOperationAdmin,
+        BuildAdmin as BaseBuildAdmin,
+        CategoryAdmin as BaseCategoryAdmin,
+    )
+    from openwisp_firmware_upgrader.swapper import load_model
+
+    BatchUpgradeOperation = load_model('BatchUpgradeOperation')
+    Build = load_model('Build')
+    Category = load_model('Category')
+    DeviceFirmware = load_model('DeviceFirmware')
+    FirmwareImage = load_model('FirmwareImage')
+    UpgradeOperation = load_model('UpgradeOperation')
+
+    admin.site.unregister(BatchUpgradeOperation)
+    admin.site.unregister(Build)
+    admin.site.unregister(Category)
+
+    class BatchUpgradeOperationAdmin(BaseBatchUpgradeOperationAdmin):
+        # add your changes here
+
+    class BuildAdmin(BaseBuildAdmin):
+        # add your changes here
+
+    class CategoryAdmin(BaseCategoryAdmin):
+        # add your changes here
+
+11. Create root URL configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please refer to the `urls.py <https://github.com/openwisp/openwisp-firmware-upgrader/tree/master/tests/openwisp2/urls.py>`_
+file in the test project.
+
+For more information about URL configuration in django, please refer to the
+`"URL dispatcher" section in the django documentation <https://docs.djangoproject.com/en/dev/topics/http/urls/>`_.
+
+12. Create celery.py
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Just noticed this: 
```
12. Create celery.py
~~~~~~~~~~~~~~~~
```
`~~~~~~~~~~~~~~~~~` length changed! :smile:

Reply:
Addressed, thanks